CharacterCreation/00_New_Player_Walkthrough.txt

```diff
@@ -0,0 +1,47 @@
+## Step 0: Ask for Advice
+No really, if you get stuck at any point or just want a second opinion, other players can be tremendously helpful and
+are usually more than happy to assist new players getting their feet wet.
+
+## Step 1: Race
+*Section to be added on account of impending large-scale racial rework*
+
+## Step 2: Class
+For players new to role-playing games we recommend the Soldier or Hitman classes (and possibly Pilot).
+*Section to be expanded later on account of impending large-scale class rework*
+
+## Step 3: Background and Skills
+*Section to be added on account of impending addition of this feature*
+
+## Step 4: Attribute Point Allocation
+Now that you know what your character is capable of in-combat (your class) and out-of-combat (your non-combat skills), 
```

Comment:
I know it's an intentional and helpful oversimplification, but I don't like saying that your class is for in-combat and you skills are for out-of-combat.

Reply:
Found a way to restate.